### DISCUSSION TITLE
loking for vue img slider

### DISCUSSION THREAD

**Discussion by YaelOushky** [community/community#65609] [Other Feature Feedback, Questions, & Ideas]
loking for vue img slider

**Comment by ShonChassel** [comment]
hey 
chek this 
https://www.npmjs.com/package/vue-easy-slider

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
